tests/test_model.py

```diff
@@ -7821,6 +7821,22 @@ def assert_isbns(expected, result_query):
         updated_isbns = self.collection.isbns_updated_since(self._db, timestamp)
         assert_isbns([i1], updated_isbns)
 
+    def test_custom_lists(self):
+        # A Collection can be associated with one or more CustomLists.
+        list1, ignore = get_one_or_create(self._db, CustomList, name=self._str)
+        list2, ignore = get_one_or_create(self._db, CustomList, name=self._str)
+        self.collection.customlists = [list1, list2]
+        eq_(0, len(list1.entries))
+        eq_(0, len(list2.entries))
+
+        # When a new pool is added to the collection and its presentation edition is
+        # calculated for the first time, it's automatically added to the lists.
+        work = self._work(collection=self.collection, with_license_pool=True)
+        eq_(1, len(list1.entries))
+        eq_(1, len(list2.entries))
+        eq_(work, list1.entries[0].work)
+        eq_(work, list2.entries[0].work)
+
```

Comment:
Suggest you also add a test that verifies nothing happens when a Work's presentation edition changes or a LicensePool changes works.